modules/living_spaces_group/living_spaces_group.api.php

```diff
@@ -201,6 +201,20 @@ function hook_living_spaces_group_exclude_permissions() {
   return $permissions;
 }
 
+/**
+ * Provide a list of space UUIDs to remove their memberships.
+ *
+ * @return array
+ *   An array of space UUIDs.
+ */
+function hook_living_spaces_group_default_spaces() {
+  $uuids = [];
+
+  $uuids[] = '12345678-abcd-abcd-abcd-12345678abcd';
+
+  return $uuids;
+}
```

Comment:
This is not needed. All space templates have 'is_default' field which marks those as templates